Package.swift

```diff
@@ -19,7 +19,9 @@ let package = Package(
         .package(url: "https://github.com/apple/swift-nio.git", from: "2.0.0"),
         .package(url: "https://github.com/apple/swift-atomics.git", from: "1.0.2"),
         .package(url: "https://github.com/apple/swift-metrics.git", from: "2.4.1"),
-        .package(url: "https://github.com/slashmo/swift-w3c-trace-context.git", from: "1.0.0-beta.1"),
+        //.package(url: "https://github.com/slashmo/swift-w3c-trace-context.git", from: "1.0.0-beta.1"),
+        //.package(name: "swift-w3c-trace-context", path: "../swift-w3c-trace-context"),
+        .package(url: "https://github.com/t089/swift-w3c-trace-context.git", branch: "trace-id-bytes"),
```

Comment:
Please see https://github.com/slashmo/swift-w3c-trace-context/pull/18 for details on this